components/expofp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/expofp",
-  "version": "0.0.2",
+  "version": "0.1.0",
```

Comment:
What kind of changes would warrant updating the minor version instead of the patch version?

I try to follow [these guidelines as explained in the Pipedream docs](https://pipedream.com/docs/components/guidelines/#versioning) when it comes to updating the version number. Since there are new components added to the app and not just bugfixes, I updated the minor version.